src/components/InitiativeCard/InitiativeCard.styles.ts

```diff
@@ -1,18 +1,18 @@
 import { cnb } from 'cnbuilder';
 
-export const root = 'su-group su-relative su-w-full su-h-full';
+export const root = 'su-group su-relative su-w-full su-h-full su-@container';
 
-export const topWrapper = 'su-relative';
+export const topWrapper = 'su-relative @xs:su-text-18 @sm:su-text-21 @md:su-text-23';
```

Comment:
Use container query in the parent container to set base font size, which means our breakpoint based modular typography become container size based

Reply:
The initiative cards gets narrow at some breakpoints, also the heading font is very wide which makes container query very useful as progressive enhancement.